### DISCUSSION TITLE
Time controlled pick-up /delivery

### DISCUSSION THREAD

**Discussion by funmusic73** [Courseplay/courseplay#6876] [Ideas]
Hello 

would it be feasible to have certain devices ("feed carts", "yard cleaners", water barrels, etc.) driven over courses in a time-controlled way? Could then perhaps be with the option every 1-8 hours (playing time).

That came to mind recently when cleaning the stables.

Best regards Frank

**Comment by Tensuko** [comment]
This might be possible, when we change the Waittime use, maybe we can get it to work with this. See also #6402
